go.mod

```diff
@@ -0,0 +1,118 @@
+	github.com/mitchellh/cli v1.0.0
+	github.com/onsi/ginkgo v1.10.3
+	github.com/onsi/gomega v1.7.1
+	github.com/openshift-metal3/terraform-provider-ironic v0.1.8
+	github.com/openshift/api v3.9.1-0.20191014195513-c9253efc14f4+incompatible
+	github.com/openshift/client-go v0.0.0-20191001081553-3b0e988f8cb0
+	github.com/openshift/cloud-credential-operator v0.0.0-20190905120421-44ed18ef8496
+	github.com/openshift/cluster-api v0.0.0-20190923092624-4024de4fa64d
+	github.com/openshift/cluster-api-provider-gcp v0.0.0-20190826205919-0cd5daa07e0d
+	github.com/openshift/cluster-api-provider-libvirt v0.0.0-20190613141010-ecea5317a4ab
+	github.com/openshift/library-go v0.0.0-20191003152030-97c62d8a2901
+	github.com/openshift/machine-config-operator v0.0.0-20191029224459-e8050812d506
+	github.com/pborman/uuid v1.2.0
+	github.com/pkg/errors v0.8.1
+	github.com/pkg/sftp v1.10.0
+	github.com/shurcooL/httpfs v0.0.0-20190707220628-8d4bc4ba7749
+	github.com/shurcooL/vfsgen v0.0.0-20181202132449-6a9ea43bcacd
+	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/cobra v0.0.5
+	github.com/stoewer/go-strcase v1.1.0 // indirect
+	github.com/stretchr/testify v1.4.0
+	github.com/terraform-providers/terraform-provider-aws v0.0.0-00010101000000-000000000000
+	github.com/terraform-providers/terraform-provider-azurerm v1.37.0
+	github.com/terraform-providers/terraform-provider-google v0.0.0-20191204001720-5937ed8454f5
+	github.com/terraform-providers/terraform-provider-ignition v1.1.0
+	github.com/terraform-providers/terraform-provider-local v1.4.0
+	github.com/terraform-providers/terraform-provider-openstack v1.24.0
+	github.com/terraform-providers/terraform-provider-random v2.0.0+incompatible
+	github.com/vincent-petithory/dataurl v0.0.0-20160330182126-9a301d65acbb
+	golang.org/x/crypto v0.0.0-20191202143827-86a70503ff7e
+	golang.org/x/lint v0.0.0-20191125180803-fdd1cda4f05f
+	golang.org/x/net v0.0.0-20191204025024-5ee1b9f4859a // indirect
+	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
+	golang.org/x/sys v0.0.0-20191128015809-6d18c012aee9
+	golang.org/x/tools v0.0.0-20191204011308-9611592c72f6 // indirect
+	golang.org/x/xerrors v0.0.0-20191011141410-1b5146add898 // indirect
+	google.golang.org/api v0.13.0
+	gopkg.in/AlecAivazis/survey.v1 v1.8.7
+	gopkg.in/ini.v1 v1.42.0
+	gopkg.in/yaml.v2 v2.2.7
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
+	k8s.io/klog v1.0.0
+	k8s.io/utils v0.0.0-20191114184206-e782cd3c129f
+	sigs.k8s.io/cluster-api-provider-aws v0.0.0-00010101000000-000000000000
+	sigs.k8s.io/cluster-api-provider-azure v0.0.0-00010101000000-000000000000
+	sigs.k8s.io/cluster-api-provider-openstack v0.0.0
+)
+
+replace (
+	github.com/Azure/azure-sdk-for-go => github.com/hashicorp/azure-sdk-for-go v36.2.2-hashi+incompatible
+	github.com/coreos/ignition => github.com/coreos/ignition v0.28.0
+	github.com/go-log/log => github.com/go-log/log v0.1.1-0.20181211034820-a514cf01a3eb
```

Comment:
this is not in the require block?

Reply:
it's a dep of a dep that gets pulled in during go vet if I'm not mistaken. Need to make sure it's a compatible version.